include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_pressure_correction.cpp

```diff
@@ -318,19 +318,35 @@ void
 TimeIntBDFPressureCorrection<dim, Number>::do_timestep_solve()
 {
   // perform the sub-steps of the pressure-correction scheme
-
-  momentum_step();
+  if(this->update_velocity)
+  {
+    momentum_step();
+  }
 
   VectorType pressure_increment;
-  pressure_increment.reinit(pressure_np, false /* init with zero */);
-
-  pressure_step(pressure_increment);
+  if(this->update_pressure)
+  {
+    pressure_increment.reinit(pressure_np, false /* omit_zeroing_entries */);
+    pressure_step(pressure_increment);
+  }
```

Comment:
A similar question as here: https://github.com/exadg/exadg/pull/574/files#r1860844732

If `update_velocity == false` and `update_pressure == true` the coupling "structure -> fluid" would only be via the velocity DBC entering the pressure rhs vector, correct?